nilearn/decoding/decoder.py

```diff
@@ -1,43 +1,48 @@
 """High-level decoding object that exposes standard classification and
 regression strategies such as SVM, LogisticRegression and Ridge, with optional
-feature selection, integrated hyper-parameter selection
-and aggregation strategy in which the best models
-within a cross validation loop are averaged.
+feature selection, integrated hyper-parameter selection and aggregation strategy
```

Comment:
Line limit should be 79 not 80, see: https://www.python.org/dev/peps/pep-0008/#id19